opencog/opencog/spacetime/atom_types.script

```diff
@@ -15,6 +15,7 @@ IMAGINARY_STRUCTURE_NODE <- OBJECT_NODE
 TIME_NODE <- NODE
 TIME_DOMAIN_NODE <- NODE
 AT_TIME_LINK <- ORDERED_LINK
+AT_LOCATION_LINK <- ORDERED_LINK
```

Comment:
never mind, I just looked its not as much as I thought. @ngeiswei would you care to review the 3D code and git-pull it into the main opencog repo? or should I?